src/libraries/Pool.sol

```diff
@@ -455,10 +456,10 @@ library Pool {
             // negation safe as amount0 and amount1 are always positive
             delta = toBalanceDelta(-(amount0.toInt128()), -(amount1.toInt128()));
             if (amount0 > 0) {
```

Comment:
would love a quick comment on each use saying why its safe to do it like this. Just 1 quick sentence nothing complicated :)

Reply:
e.g.
```
// FullMath.mulDiv not needed because numerator cannot exceed `type(int128).max) * Q128`
```
or something like that?
and then something similar for swap function too